internal/storage/v2/clickhouse/config/config.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"runtime"
+	"time"
+)
+
+// Configuration holds all configuration options for using ClickHouse as storage.
+// ClickhouseConfig configures the clickhouse-go/v2 client.
+// ChPoolConfig chpool configuration.
+type Configuration struct {
+	ClickhouseConfig ClickhouseConfig
+	ChPoolConfig     ChPoolConfig
+}
+
+func DefaultConfiguration() Configuration {
+	return Configuration{
+		DefaultClickhouseConfig(),
+		DefaultChPoolConfig(),
+	}
+}
+
+// ClickhouseConfig holds clickhouse-go/v2 client connection configuration for reading traces.
+// more detail see:https://clickhouse.com/docs/integrations/go#connection-settings
+type ClickhouseConfig struct {
```

Comment:
structs are defined at the top of the file before any functions. Constants are defined above structs.